packages/lesswrong/lib/instanceSettings.ts

```diff
@@ -171,3 +171,6 @@ export const ckEditorWebsocketUrlOverrideSetting = new PublicInstanceSetting<str
 export const testServerSetting = new PublicInstanceSetting<boolean>("testServer", false, "warning")
 
 export const disableEnsureIndexSetting = new PublicInstanceSetting<boolean>("disableEnsureIndex", false, "optional");
+
+export const lowKarmaUserVotingCutoffDateSetting = new PublicInstanceSetting<string>("lowKarmaUserVotingCutoffDate", "11-30-2022", "optional");
+export const lowKarmaUserVotingCutoffKarmaSetting = new PublicInstanceSetting<number>("lowKarmaUserVotingCutoffKarma", 10, "optional");
```

Comment:
I would either make this null by default or signpost it that it's currently LW-only.

(Sorry for reviewing your draft, I was curious)

Reply:
Signposted :)